spec/example_app/config/database.yml

```diff
@@ -24,6 +24,7 @@ default: &default
 development:
   <<: *default
   database: example_app_development
+  host: localhost
```

Comment:
Hrm, why is this necessary? I thought localhost was the default?

Reply:
I think this was one of the things I tried when getting postgres working again, but it turns out it wasn't the thing that actually worked, so I'll close this.